src/hotspot/os_cpu/linux_arm/thread_linux_arm.hpp

```diff
@@ -40,26 +40,10 @@
   frame pd_last_frame();
 
  public:
-  intptr_t* last_Java_fp()                       { return _anchor.last_Java_fp(); }
-  void  set_last_Java_fp(intptr_t* fp)           { _anchor.set_last_Java_fp(fp);  }
-  void  set_last_Java_pc(address pc)             { _anchor.set_last_Java_pc(pc);  }
-
```

Comment:
Why is this remove from _arm, but not for thread_linux_aarch64.hpp

Reply:
Oops, I didn't mean to remove these ones.

I was focusing on the stack related functions. but I can file an RFE to remove these next.  Thanks!